cypress/e2e/signIn.cy.js

```diff
@@ -4,31 +4,36 @@
 import SignInPageObject from '../support/pages/signIn.pageObject';
 import HomePageObject from '../support/pages/home.pageObject';
 
-const signInPage = new SignInPageObject();
-const homePage = new HomePageObject();
+const SignInPage = new SignInPageObject();
+const HomePage = new HomePageObject();
 
 describe('Sign In page', () => {
   let user;
 
   before(() => {
-    cy.task('db:clear');
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
     });
   });
 
   it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
+    SignInPage.visit();
     cy.register(user.email, user.username, user.password);
 
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
+    SignInPage.typeEmail(user.email);
+    SignInPage.typePassword(user.password);
+    SignInPage.clickSignInBtn();
 
-    homePage.assertHeaderContainUsername(user.username);
+    HomePage.assertHeaderContainUsername(user.username);
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    SignInPage.visit();
 
+    SignInPage.typeEmail('testtesttest@example.com');
+    SignInPage.typePassword('test123123!');
+    SignInPage.clickSignInBtn();
+
+    cy.contains('email or password:is invalid').should('be.visible');
```

Comment:
Ensure that the error message 'email or password:is invalid' is exactly what appears on the page. If the actual message differs, the test will fail. Consider verifying the exact message or using a partial match if the message might change.